frontend/src/ts/guideConst.ts

```diff
@@ -141,12 +152,6 @@ do {
     var configuration = try ClickstreamAnalytics.getClickstreamConfiguration()
     configuration.appId = "${TEMPLATE_APP_ID}"
     configuration.endPoint = "${TEMPLATE_SERVER_ENDPOINT}"
-    configuration.authCookie = "your authentication cookie"
-    configuration.sessionTimeoutDuration = 1800000
-    configuration.isTrackAppExceptionEvents = false
-    configuration.isLogEvents = true
-    configuration.isCompressEvents = true    
-    configuration.isLogEvents = true
```

Comment:
The entire configuration section should be removed from both Android and iOS.

Reply:
Because we were auto replace the `AppId` and `Endpoint`  in the configuration code. I think this is useful. What do you think ? @luorobin-a2z 
<img width="764" alt="image" src="https://github.com/awslabs/clickstream-analytics-on-aws/assets/7011985/de517cb4-284d-4a67-90ad-040197e55bd5">

Reply:
`appId` and `endpoint` are included in the `amplifyconfiguration.json` file. This step is not necessary for integrating the SDK. After discussing with @luorobin-a2z , and decided to delete this part. Users can view more detailed usage configuration in the doc link below.

Reply:
OK